pkgs/development/python-modules/textual/default.nix

```diff
@@ -48,6 +48,8 @@ buildPythonPackage rec {
     typing-extensions
   ];
 
+  doCheck = false; # broken with rich 13.3.0
```

Comment:
Please also add why

Reply:
I've updated textual instead (which required a markdown-it-py update), now tests pass.